test/e2e/page-objects/identity/appellantDetails.js

```diff
@@ -111,6 +111,24 @@ function enterAppellantContactDetailsAndContinue(commonContent, language) {
   I.click(commonContent.continue);
 }
 
+function enterAppellantContactDetailsAndContinueAfterSignIn(commonContent, language) {
```

Comment:
Again original method is here https://github.com/hmcts/sscs-submit-your-appeal/blob/4f7ace0242ee6c81d5d5e2f2ef421cf8f2e7a70b/test/e2e/page-objects/identity/appellantDetails.js#L96

don't need this duplicate method